spec/features/adding_cheet_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+feature "listing all cheets" do
+
+  before(:each) {
```

Comment:
if doing multi-line please prefer "do ... end" over {} and notice that you don't need the :each you can just have 

```rb
before { ... }
```

or 

```rb
before do 
  ... 
end
```